### DISCUSSION TITLE
Unit testing : how to check header value ?

### DISCUSSION THREAD

**Discussion by cmarmonier** [Kong/insomnia#6206] [Help]
Hello,

I want to make an unit test. I do it : 
```
const response1 = await insomnia.send();
expect(response1.status).to.equal(200);
```

Now I want to test that the value of "Location" header parameter is "xxx".
How cant I do it ?
Where is documentation for insomnia.send object (I didn't find it) ?

Also for an another request, I want to verify that "Location" header parameter do not exist

I thank you

**Comment by x80486** [comment]
Run into this one today; this is how it's done: `expect(response.headers["Location"]).to.equal("Your_Expected_Value_Here");`.

If you run into something similar in the future, open up `View > Toggle DevTools`, and log what you are looking for using `console.log`. For instance:

```javascript
const response = await insomnia.send();
console.log(response.headers);
```
